code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -168,7 +168,7 @@
 			target.apply_armoured_damage(get_xeno_damage_slash(target, xeno.caste.melee_damage_upper), ARMOR_MELEE, BRUTE, rand_zone())
 			playsound(get_turf(target), 'sound/weapons/alien_claw_flesh4.ogg', 30, TRUE)
 			xeno.flick_heal_overlay(1 SECONDS, "#00B800")
-			xeno.gain_health(30)
+			xeno.gain_health(get_xeno_damage_slash(target, xeno.caste.melee_damage_upper))
```

Comment:
Just revert this; I don't think it necessary for now.

Reply:
done